### PR TITLE
small code cleanup

### PR DESCRIPTION
**Pull request by HSJ-OSRS**
not sure if this is helpful or not but though id do it anyway

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java [hunk 1]
```diff
@@ -286,7 +252,6 @@ public void onMenuOptionClicked(MenuOptionClicked event)
 			|| event.getId() == ItemID.SUPERANTIPOISON4))
 		{
 			createGameTimer(SUPERANTIPOISON);
-			return;
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java]: :-1: If ever more code is added below this check, it'd be easier to leave this statement in than to expect the new code re-add it.

File: runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/EmoteClue.java [hunk 2]
```diff
@@ -30,355 +30,10 @@
 import java.util.Set;
 import lombok.Getter;
 import net.runelite.api.EquipmentInventorySlot;
-import static net.runelite.api.EquipmentInventorySlot.AMULET;
-import static net.runelite.api.EquipmentInventorySlot.BODY;
-import static net.runelite.api.EquipmentInventorySlot.BOOTS;
-import static net.runelite.api.EquipmentInventorySlot.CAPE;
-import static net.runelite.api.EquipmentInventorySlot.GLOVES;
-import static net.runelite.api.EquipmentInventorySlot.HEAD;
+import static net.runelite.api.EquipmentInventorySlot.*;
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/EmoteClue.java]: Is every slot being imported before this commit? It doesn't look like it.

Anyway, it looks like you missed removing the `LEGS` import. :question:

File: runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/EmoteClue.java [hunk 3]
```diff
@@ -30,355 +30,10 @@
...
-import static net.runelite.api.ItemID.VERACS_PLATESKIRT_100;
-import static net.runelite.api.ItemID.WHITE_APRON;
-import static net.runelite.api.ItemID.WHITE_BOATER;
-import static net.runelite.api.ItemID.WHITE_HEADBAND;
-import static net.runelite.api.ItemID.ZAMORAK_CROZIER;
-import static net.runelite.api.ItemID.ZAMORAK_FULL_HELM;
-import static net.runelite.api.ItemID.ZAMORAK_GODSWORD;
-import static net.runelite.api.ItemID.ZAMORAK_MITRE;
-import static net.runelite.api.ItemID.ZAMORAK_STOLE;
+import static net.runelite.api.ItemID.*;
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/EmoteClue.java]: Please no.

File: runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/EmoteClue.java [hunk 4]
```diff
@@ -387,34 +42,8 @@
 import net.runelite.client.plugins.cluescrolls.clues.emote.AllRequirementsCollection;
 import net.runelite.client.plugins.cluescrolls.clues.emote.AnyRequirementCollection;
 import net.runelite.client.plugins.cluescrolls.clues.emote.Emote;
-import static net.runelite.client.plugins.cluescrolls.clues.emote.Emote.ANGRY;
-import static net.runelite.client.plugins.cluescrolls.clues.emote.Emote.BECKON;
-import static net.runelite.client.plugins.cluescrolls.clues.emote.Emote.BLOW_KISS;
-import static net.runelite.client.plugins.cluescrolls.clues.emote.Emote.BOW;
+import static net.runelite.client.plugins.cluescrolls.clues.emote.Emote.*;
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/EmoteClue.java]: Same as the equipment slot change above.

**Comment by forsco** [comment]
I thought we didn't do wild card imports?

**Comment by Nightfirecat** [comment]
I can't speak much to the general optimizations, but they seem fine to me.

As for the imports change commit, [most of your changes don't follow this project's code conventions](https://github.com/runelite/runelite/wiki/Code-Conventions#imports). Beyond that, in some files you chose to collapse multiple imports into a wildcard, and in others you did the opposite. Is there a reason for that?

**Comment by HSJ-OSRS** [comment]
for the imports i just ran the automated import optimise thing in idea wasn't really sure how well it would do i guess it was bad.

**Pull request merged**